src/test/java/seedu/address/logic/commands/LabelCommandTest.java

```diff
@@ -0,0 +1,112 @@
+package seedu.address.logic.commands;
+
+import static org.junit.Assert.assertEquals;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.LabelCommand.DEFAULT_FILENAME;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_MEDICINE;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_MEDICINE;
+import static seedu.address.testutil.TypicalMedicines.getTypicalInventory;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.text.PDFTextStripper;
+import org.apache.pdfbox.text.PDFTextStripperByArea;
+
+import org.junit.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.FileName;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.medicine.Medicine;
+
+public class LabelCommandTest {
+
+    private Model model = new ModelManager(getTypicalInventory(), new UserPrefs());
+    private CommandHistory commandHistory = new CommandHistory();
+    private FileName fileName = new FileName(DEFAULT_FILENAME);
+
+    @Test
+    public void execute_labelMedicineAtIndexOneDefaultFileNameUnfilteredList_success()
+            throws IOException, CommandException {
+        Medicine medicineToLabel = model.getFilteredMedicineList().get(0);
+
+        LabelCommand labelCommand = new LabelCommand(INDEX_FIRST_MEDICINE, new FileName(DEFAULT_FILENAME));
+        labelCommand.execute(model, commandHistory);
+        File printedFile = new File("./PDF/" + "to_print.pdf");
```

Comment:
not a good idea to use `.` for current directory. consider getting the full file path from the system instead